linkerd/setup.py

```diff
@@ -27,7 +27,7 @@ def get_dependencies():
         return f.readlines()
 
 
-CHECKS_BASE_REQ = 'datadog-checks-base'
+CHECKS_BASE_REQ = 'datadog-checks-base>=11.0.0'
```

Comment:
Are you on latest master? I noticed that the latest update the openmetrics template is not here `ignore_metrics_by_label`
```suggestion
CHECKS_BASE_REQ = 'datadog-checks-base>=15.4.0'
```
will need to be bumped due to base class changes

Reply:
Indeed!